xml_converter/generators/proto_templates/node.proto

```diff
@@ -0,0 +1,274 @@
+syntax = "proto3";
+import "google/protobuf/any.proto";
```

Comment:
Referencing the later comment. `Any` should not be necessary so we probably don't need this.

Reply:
Removed.